scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -35,6 +36,27 @@ public class CompileTargets
             "vbc.exe"
         };
 
+        public static readonly string[] NetCoreAppRids = new[]
+        {
+            "win7-x64",
+            "win7-x86",
+            "osx.10.10-x64",
+            "osx.10.11-x64",
+            "ubuntu.14.04-x64",
+            "centos.7-x64",
+            "rhel.7-x64",
+            "rhel.7.2-x64",
+            "debian.8.2-x64",
```

Comment:
I tried to find where I got the debian 8.2 rid, but can't find it anymore. 

After @MichaelSimons confirms, will remove it.